### DISCUSSION TITLE
[Script request] InvoiceNinja

### DISCUSSION THREAD

**Discussion by uSlackr** [community-scripts/ProxmoxVE#98] [Request script]
### Application Name

InvoiceNinja

### Website

https://www.invoiceninja.org/

### Description

Self-hosted requires a Mariadb or mysql db instance.  Not sure how these scripts handle that.  Willing to help test as we go.

There is a Turnkey instance, but 1) it appears months behind current release and 2) Did the things but I'm not getting my account approved.  

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
#Blocked

In Pipeline by me.

**Reply by shrijayan** [reply]
Any updates on this needed very much! Thanks in advance.

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1197

InvoiceNinja is done at DEV - please test it carefully and give feedback in linked issue. 
An **important test** point should be the update, as I'm not entirely sure whether I've backed everything up correctly, since the InvoiceNinja documentation doesn't provide much output. 

```bash -c "$(curl -fsSL https://github.com/community-scripts/ProxmoxVED/raw/main/ct/invoiceninja.sh)"```
First visit http://IP:8080/setup to complete the initial configuration
Database credentials are stored in ~/invoiceninja.creds
Configure SMTP settings in /opt/invoiceninja/.env for email functionality

Note: This is not in the official repo yet—it's just a dev version! After merging into ProxmoxVE, it will need to be recreated.

**Comment by uSlackr** [comment]
@MickLesk Thanks so much for this.  I didn't have a chance to test during beta as my proxmox was a mess.  Got some help on the Discord and all is healthy.  I am testing now.  
I initially had an issue with the 500 server issue (using latest script).  I ran the chmod from the closed issue and that resolved it, but I'm not sure that maybe it just needed time.  I will re-test setup.  But, so far, testing is going well.
